public/partials/shortcodes/process/process_form_submission.php

```diff
@@ -177,8 +177,12 @@
 	// Check the opt-in value - is it double or single?
 	// Double opt-in means 'status_if_new' => 'pending'
 	$double_optin = isset( $optin_settings['optin'] ) ? (int) $optin_settings['optin'] : 0;
+        $was_unsubscribed = isset($member_exists['status']) && $member_exists['status'] == 'unsubscribed';
 
-	if ( $double_optin === 1 ) {
+        // If the user was unsubscribed and is re-subscribing, we set the status to 'pending', which
+        // causes Mailchimp to send them a confirmation email.  This is the only way Mailchimp will
+        // allow us to re-subscribe the user.
```

Comment:
I would recommend moving this comment to just above where `$was_unsubscribed` is defined.